### ISSUE TITLE
install erro: object ‘clusterApply’ is not exported by 'namespace:BiocGenerics'

### ISSUE THREAD

**Issue by chairchild** [cole-trapnell-lab/monocle3#552]
when i installed monocle3, with "devtools::install_github("cole-trapnell-lab/monocle-release", ref="monocle3_alpha")"

erro occured:
Error: object ‘clusterApply’ is not exported by 'namespace:BiocGenerics'
Execution halted
ERROR: lazy loading failed for package ‘monocle’

**Comment by chairchild** [comment]
additionally, when i use "devtools::install_github('cole-trapnell-lab/monocle3')", it can installed correctely.

**Comment by ixxmu** [comment]
yes, it's a bug on 2.20.0 version, you can find the updated version on monocle2 archive page **[here]**(https://www.bioconductor.org/packages/3.14/bioc/src/contrib/Archive/monocle/), its **2.21.1**, after installing it manually, everything is back to normal!
